### ISSUE TITLE
influx_transfer_files2db does not work

### ISSUE THREAD

**Issue by 8guawong** [TotallyMehis/Influx-Timer#95]
first of all 

https://github.com/InfluxTimer/sm-timer/blob/cabc66a2422e5be380d720a5054d6ea07ebcac08/addons/sourcemod/scripting/influx_transfer_files2db.sp#L25

when you do this you can only use the command from server 

so why not just RegServerCmd?

2nd of all 

https://github.com/InfluxTimer/sm-timer/blob/cabc66a2422e5be380d720a5054d6ea07ebcac08/addons/sourcemod/scripting/influx_transfer_files2db.sp#L131

https://github.com/InfluxTimer/sm-timer/blob/cabc66a2422e5be380d720a5054d6ea07ebcac08/addons/sourcemod/scripting/influx_transfer_files2db.sp#L167

after SQL_EscapeString everything is just messed up

following is an example output after the SQL_EscapeString

add LogMessage(szData);  after SQL_EscapeString and get the following

[influx_transfer_files2db.smx] \"\"\\\"\\\\\\\"\\\\\\\\\\\\\\\"\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\"\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\"\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\"

so basically its impossible to import runs and zones from files

hope everything is clearly stated and understandable

and you can fix this problem 

really wants zones and runs in mysql instead of files 

Thanks!

**Comment by TotallyMehis** [comment]
Fixed in 2beef4e
